readme.md

```diff
@@ -45,6 +48,13 @@ String to use as replacement for reserved filename characters.
 
 Cannot contain: `<` `>` `:` `"` `/` `\` `|` `?` `*`
 
+### condenseReplacements
+
+Type: `boolean`\
+Default: `false`
+
+Preserves the repeated replacements of a given string.
```

Comment:
This sentence is inconsistent compared to the other ones. And it's also incorrect. It doesn't describe what the option does, but rather what it *doesn't do*.